lib/school_house_web/templates/page/index.html.heex

```diff
@@ -245,7 +245,7 @@
 <div class="bg-body dark:bg-body-dark">
```

Comment:
Shouldn't be`<section>` ?

Reply:
Our current design may have too many `div`s but to get a general idea, here's an explanation about `section` and headings:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section